### DISCUSSION TITLE
Simulation Processes Tab

### DISCUSSION THREAD

**Discussion by pburkat** [Open-Systems-Pharmacology/Forum#1268] [Questions & Problems]
Hello, I am running into a problem with my simulation for ketamine, I don't see a reaction diagram and when I configure the simulation, the Processes tab page is completely blank, I believe these 2 issues are connected. Any suggestions?

**Comment by msevestre** [comment]
Have you created a link between your enzyme and your processes? Can you show the process tab?

**Reply by pburkat** [reply]
I think that's the problem, none of my compounds/enzymes are on the process tab. There's nothing to select.
[Image.pdf](https://github.com/Open-Systems-Pharmacology/Forum/files/9210186/Image.pdf)

**Reply by msevestre** [reply]
Have you added the expression profile to your Individual? 
Expression profile are now building blocks since v10 that need to be added to an individual
Please check the documentation for more details
https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-creating-individuals#expression

**Reply by pburkat** [reply]
Yes, that must be it. I'll try that thanks.

**Reply by pburkat** [reply]
Now it works, thanks.
